Source/stores.cpp

```diff
@@ -1827,7 +1826,7 @@ bool __fastcall IdItemOk(ItemStruct *i)
 	if ( i->_itype != -1 )
 	{
 		if ( i->_iMagical )

```

Comment:
LGTM.

If you want to add another enum, the definition for `_iMagical` looks something as follows (I could not find it in the SYM):

```c
// Item quality levels.
typedef enum {
	ITEM_QUALITY_NORMAL = 0,
	ITEM_QUALITY_MAGIC  = 1,
	ITEM_QUALITY_UNIQUE = 2,
} item_quality;
```